src/styleService.ts

```diff
@@ -78,6 +78,21 @@ export class StyleService {
 
         this.updateStyles(styles);
 
+        // console.log(styles);
```

Comment:
Tony, please remove unnecessary comments like this `// console.log(styles);`

Reply:
Oops I forgot.